dom/src/MainDOMSource.ts

```diff
@@ -111,18 +116,18 @@ export class MainDOMSource implements DOMSource {
     };
   }
 
-  public elements(): MemoryStream<Element> {
-    let output$: Stream<Element | Array<Element>>;
+  public elements(): MemoryStream<Array<Element>> {
+    let output$: Stream<Array<Element>>;
     if (this._namespace.length === 0) {
-      output$ = this._rootElement$;
+      output$ = this._rootElement$.map(Array.of);
```

Comment:
`Array.of` is ES6, and I suspect this is what is breaking tests on IE10. We could just use `.map(x => [x])` instead.

Reply:
ah, yes of course